src/main/resources/swagger/api-docs.yaml

```diff
@@ -1220,6 +1342,17 @@ definitions:
         additionalProperties:
           type: string
         description: Map of attributes
+  MethodConfigACL:
+    properties:
+      userId:
+        type: string
+        description: a user's ID or "public"
+      accessLevel:
+        type: string
+        description: one of "NO ACCESS", "READER", or "OWNER"
+    required:
+      - userId
+      - accessLevel
```

Comment:
this file did not work for me when I loaded swagger locally; you can see the syntax problems I found above. Please test locally!